instrumentation/hystrix-1.4/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/hystrix/HystrixObservableTest.java

```diff
@@ -0,0 +1,528 @@
+                new Parameter(
+                    null, Schedulers.trampoline(), cmd -> cmd.observe().toBlocking().first()))),
+        Arguments.of(
+            named(
+                "observe+C",
+                new Parameter(
+                    null, Schedulers.computation(), cmd -> cmd.observe().toBlocking().first()))),
+        Arguments.of(
+            named(
+                "observe+IO",
+                new Parameter(null, Schedulers.io(), cmd -> cmd.observe().toBlocking().first()))),
+        Arguments.of(
+            named(
+                "observe+NT",
+                new Parameter(
+                    null, Schedulers.newThread(), cmd -> cmd.observe().toBlocking().first()))));
+  }
+
+  private static Stream<Arguments> provideCommandActionArguments() {
+    return Stream.concat(
+        baseArguments(),
+        Stream.of(
+            Arguments.of(
+                named(
+                    "toObservable block",
+                    new Parameter(
+                        Schedulers.computation(),
+                        Schedulers.newThread(),
+                        cmd -> {
+                          BlockingQueue<String> queue = new LinkedBlockingQueue<>();
+                          Subscription subscription =
+                              cmd.toObservable()
+                                  .subscribe(
+                                      next -> {
+                                        try {
+                                          queue.put(next);
+                                        } catch (InterruptedException e) {
+                                          throw new RuntimeException(e);
+                                        }
+                                      });
+                          String returnValue;
+                          try {
+                            returnValue = queue.take();
+
+                          } catch (InterruptedException e) {
+                            throw new RuntimeException(e);
+                          }
+                          subscription.unsubscribe();
+                          return returnValue;
+                        })))));
+  }
+
+  @ParameterizedTest
+  @MethodSource("provideCommandFallbackArguments")
+  void testCommandFallbacks(Parameter parameter) {
+
+    class TestCommand extends HystrixObservableCommand<String> {
+      protected TestCommand(Setter setter) {
+        super(setter);
+      }
+
+      @Override
+      protected Observable<String> construct() {
+        Observable<String> err =
+            Observable.defer(() -> Observable.error(new IllegalArgumentException()))
+                .map(
+                    error -> {
+                      if (error instanceof Throwable) {
+                        return ((Throwable) error).getMessage();
+                      } else {
+                        throw new IllegalStateException("Expected Throwable result");
+                      }
+                    })
```

Comment:
This is a new addition, not present in the original groovy test. Is this used?

Reply:
when I was writing the test I had some return type errors for this method and I wrote this to get around them, but now when I remove this code I no longer see those issues so I must have had another unrelated issue at the time that I have since resolved. I have removed the unneeded code from this PR